be/src/agent/agent_task.cpp

```diff
@@ -160,6 +160,25 @@ void run_drop_tablet_task(const std::shared_ptr<DropTabletAgentTaskRequest>& age
 
     auto dropped_tablet = StorageEngine::instance()->tablet_manager()->get_tablet(drop_tablet_req.tablet_id);
     if (dropped_tablet != nullptr) {
+        if (!config::enable_drop_tablet_if_unfinished_txn) {
+            int64_t partition_id;
+            std::set<int64_t> transaction_ids;
+            StorageEngine::instance()->txn_manager()->get_tablet_related_txns(
+                    drop_tablet_req.tablet_id, drop_tablet_req.schema_hash, dropped_tablet->tablet_uid(), &partition_id,
+                    &transaction_ids);
+            if (!transaction_ids.empty()) {
+                std::stringstream ss;
+                ss << "Failed to drop tablet when there is unfinished txns. tablet_id:" << drop_tablet_req.tablet_id;
```

Comment:
how about print the active txn ids? so that use can figure whether the long running txn is normal or not

Reply:
OK.